utils/query_builder.py

```diff
@@ -100,16 +107,31 @@ def __build_order_by(self):
         else:
             return ""
 
+    def __build_date_range(self):
```

Comment:
Sorry for repeating, this I think should go in TimeEntriesQueryBuilder.

Reply:
Ok, adjustments were done.